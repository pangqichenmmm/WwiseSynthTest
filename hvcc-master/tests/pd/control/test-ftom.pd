#N canvas 0 22 1486 780 10;
#X obj 104 17 loadbang;
#X obj 629 684 print;
#X obj 173 17 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 139 99 -14.3;
#X msg 191 99 0;
#X obj 104 54 t b b b b b b;
#X obj 629 649 ftom;
#X msg 231 99 22 3 5;
#X msg 104 99 3.7;
#X obj 686 575 + 1;
#X obj 304 86 t b b b b b b b b b b b b b b b b b b b b b b b b b b
b b b;
#X obj 655 114 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 768 200 t b;
#X obj 799 199 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 912 285 t b;
#X obj 946 287 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 1059 373 t b;
#X obj 1094 373 t b b b b b b b b b b b b b b b b b b b b b b b b b
b b b b;
#X obj 1207 459 t b;
#X obj 628 613 mtof;
#X obj 628 576 f -12;
#X text 686 78 // Note(joe): many triggers is because i don't have
[until] right now;
#X connect 0 0 5 0;
#X connect 2 0 5 0;
#X connect 3 0 6 0;
#X connect 4 0 6 0;
#X connect 5 0 8 0;
#X connect 5 1 3 0;
#X connect 5 2 4 0;
#X connect 5 4 7 0;
#X connect 5 5 10 0;
#X connect 6 0 1 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 20 1;
#X connect 10 0 20 0;
#X connect 10 1 20 0;
#X connect 10 2 20 0;
#X connect 10 3 20 0;
#X connect 10 4 20 0;
#X connect 10 5 20 0;
#X connect 10 6 20 0;
#X connect 10 7 20 0;
#X connect 10 8 20 0;
#X connect 10 9 20 0;
#X connect 10 10 20 0;
#X connect 10 11 20 0;
#X connect 10 12 20 0;
#X connect 10 13 20 0;
#X connect 10 14 20 0;
#X connect 10 15 20 0;
#X connect 10 16 20 0;
#X connect 10 17 20 0;
#X connect 10 18 20 0;
#X connect 10 19 20 0;
#X connect 10 20 20 0;
#X connect 10 21 20 0;
#X connect 10 22 20 0;
#X connect 10 23 20 0;
#X connect 10 24 20 0;
#X connect 10 25 20 0;
#X connect 10 26 20 0;
#X connect 10 27 20 0;
#X connect 10 28 11 0;
#X connect 11 0 12 0;
#X connect 11 1 12 0;
#X connect 11 2 12 0;
#X connect 11 3 12 0;
#X connect 11 4 12 0;
#X connect 11 5 12 0;
#X connect 11 6 12 0;
#X connect 11 7 12 0;
#X connect 11 8 12 0;
#X connect 11 9 12 0;
#X connect 11 10 12 0;
#X connect 11 11 12 0;
#X connect 11 12 12 0;
#X connect 11 13 12 0;
#X connect 11 14 12 0;
#X connect 11 15 12 0;
#X connect 11 16 12 0;
#X connect 11 17 12 0;
#X connect 11 18 12 0;
#X connect 11 19 12 0;
#X connect 11 20 12 0;
#X connect 11 21 12 0;
#X connect 11 22 12 0;
#X connect 11 23 12 0;
#X connect 11 24 12 0;
#X connect 11 25 12 0;
#X connect 11 26 12 0;
#X connect 11 27 12 0;
#X connect 11 28 13 0;
#X connect 12 0 20 0;
#X connect 13 0 14 0;
#X connect 13 1 14 0;
#X connect 13 2 14 0;
#X connect 13 3 14 0;
#X connect 13 4 14 0;
#X connect 13 5 14 0;
#X connect 13 6 14 0;
#X connect 13 7 14 0;
#X connect 13 8 14 0;
#X connect 13 9 14 0;
#X connect 13 10 14 0;
#X connect 13 11 14 0;
#X connect 13 12 14 0;
#X connect 13 13 14 0;
#X connect 13 14 14 0;
#X connect 13 15 14 0;
#X connect 13 16 14 0;
#X connect 13 17 14 0;
#X connect 13 18 14 0;
#X connect 13 19 14 0;
#X connect 13 20 14 0;
#X connect 13 21 14 0;
#X connect 13 22 14 0;
#X connect 13 23 14 0;
#X connect 13 24 14 0;
#X connect 13 25 14 0;
#X connect 13 26 14 0;
#X connect 13 27 14 0;
#X connect 13 28 15 0;
#X connect 14 0 20 0;
#X connect 15 0 16 0;
#X connect 15 1 16 0;
#X connect 15 2 16 0;
#X connect 15 3 16 0;
#X connect 15 4 16 0;
#X connect 15 5 16 0;
#X connect 15 6 16 0;
#X connect 15 7 16 0;
#X connect 15 8 16 0;
#X connect 15 9 16 0;
#X connect 15 10 16 0;
#X connect 15 11 16 0;
#X connect 15 12 16 0;
#X connect 15 13 16 0;
#X connect 15 14 16 0;
#X connect 15 15 16 0;
#X connect 15 16 16 0;
#X connect 15 17 16 0;
#X connect 15 18 16 0;
#X connect 15 19 16 0;
#X connect 15 20 16 0;
#X connect 15 21 16 0;
#X connect 15 22 16 0;
#X connect 15 23 16 0;
#X connect 15 24 16 0;
#X connect 15 25 16 0;
#X connect 15 26 16 0;
#X connect 15 27 16 0;
#X connect 15 28 17 0;
#X connect 16 0 20 0;
#X connect 17 0 18 0;
#X connect 17 1 18 0;
#X connect 17 2 18 0;
#X connect 17 3 18 0;
#X connect 17 4 18 0;
#X connect 17 5 18 0;
#X connect 17 6 18 0;
#X connect 17 7 18 0;
#X connect 17 8 18 0;
#X connect 17 9 18 0;
#X connect 17 10 18 0;
#X connect 17 11 18 0;
#X connect 17 12 18 0;
#X connect 17 13 18 0;
#X connect 17 14 18 0;
#X connect 17 15 18 0;
#X connect 17 16 18 0;
#X connect 17 17 18 0;
#X connect 17 18 18 0;
#X connect 17 19 18 0;
#X connect 17 20 18 0;
#X connect 17 21 18 0;
#X connect 17 22 18 0;
#X connect 17 23 18 0;
#X connect 17 24 18 0;
#X connect 17 25 18 0;
#X connect 17 26 18 0;
#X connect 17 27 18 0;
#X connect 18 0 20 0;
#X connect 19 0 6 0;
#X connect 20 0 9 0;
#X connect 20 0 19 0;
