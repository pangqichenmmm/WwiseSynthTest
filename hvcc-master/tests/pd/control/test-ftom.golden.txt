[@ 0.000] print: -12
[@ 0.000] print: -11
[@ 0.000] print: -10
[@ 0.000] print: -9.00001
[@ 0.000] print: -8
[@ 0.000] print: -7
[@ 0.000] print: -6
[@ 0.000] print: -4.99999
[@ 0.000] print: -4.00001
[@ 0.000] print: -3
[@ 0.000] print: -1.99999
[@ 0.000] print: -1
[@ 0.000] print: 0
[@ 0.000] print: 1
[@ 0.000] print: 2
[@ 0.000] print: 3
[@ 0.000] print: 4
[@ 0.000] print: 5
[@ 0.000] print: 6
[@ 0.000] print: 7.00001
[@ 0.000] print: 8
[@ 0.000] print: 9
[@ 0.000] print: 10
[@ 0.000] print: 11
[@ 0.000] print: 12
[@ 0.000] print: 13
[@ 0.000] print: 14
[@ 0.000] print: 15
[@ 0.000] print: 16
[@ 0.000] print: 17
[@ 0.000] print: 18
[@ 0.000] print: 19
[@ 0.000] print: 20
[@ 0.000] print: 21
[@ 0.000] print: 22
[@ 0.000] print: 23
[@ 0.000] print: 24
[@ 0.000] print: 25
[@ 0.000] print: 26
[@ 0.000] print: 27
[@ 0.000] print: 28
[@ 0.000] print: 29
[@ 0.000] print: 30
[@ 0.000] print: 31
[@ 0.000] print: 32
[@ 0.000] print: 33
[@ 0.000] print: 34
[@ 0.000] print: 35
[@ 0.000] print: 36
[@ 0.000] print: 37
[@ 0.000] print: 38
[@ 0.000] print: 39
[@ 0.000] print: 40
[@ 0.000] print: 41
[@ 0.000] print: 42
[@ 0.000] print: 43
[@ 0.000] print: 44
[@ 0.000] print: 45
[@ 0.000] print: 46
[@ 0.000] print: 47
[@ 0.000] print: 48
[@ 0.000] print: 49
[@ 0.000] print: 50
[@ 0.000] print: 51
[@ 0.000] print: 52
[@ 0.000] print: 53
[@ 0.000] print: 54
[@ 0.000] print: 55
[@ 0.000] print: 56
[@ 0.000] print: 57
[@ 0.000] print: 58
[@ 0.000] print: 59
[@ 0.000] print: 60
[@ 0.000] print: 61
[@ 0.000] print: 62
[@ 0.000] print: 63
[@ 0.000] print: 64
[@ 0.000] print: 65
[@ 0.000] print: 66
[@ 0.000] print: 67
[@ 0.000] print: 68
[@ 0.000] print: 69
[@ 0.000] print: 70
[@ 0.000] print: 71
[@ 0.000] print: 72
[@ 0.000] print: 73
[@ 0.000] print: 74
[@ 0.000] print: 75
[@ 0.000] print: 76
[@ 0.000] print: 77
[@ 0.000] print: 78
[@ 0.000] print: 79
[@ 0.000] print: 80
[@ 0.000] print: 81
[@ 0.000] print: 82
[@ 0.000] print: 83
[@ 0.000] print: 84
[@ 0.000] print: 85
[@ 0.000] print: 86
[@ 0.000] print: 87
[@ 0.000] print: 88
[@ 0.000] print: 89
[@ 0.000] print: 90
[@ 0.000] print: 91
[@ 0.000] print: 92
[@ 0.000] print: 93
[@ 0.000] print: 94
[@ 0.000] print: 95
[@ 0.000] print: 96
[@ 0.000] print: 97
[@ 0.000] print: 98
[@ 0.000] print: 99
[@ 0.000] print: 100
[@ 0.000] print: 101
[@ 0.000] print: 102
[@ 0.000] print: 103
[@ 0.000] print: 104
[@ 0.000] print: 105
[@ 0.000] print: 106
[@ 0.000] print: 107
[@ 0.000] print: 108
[@ 0.000] print: 109
[@ 0.000] print: 110
[@ 0.000] print: 111
[@ 0.000] print: 112
[@ 0.000] print: 113
[@ 0.000] print: 114
[@ 0.000] print: 115
[@ 0.000] print: 116
[@ 0.000] print: 117
[@ 0.000] print: 118
[@ 0.000] print: 119
[@ 0.000] print: 120
[@ 0.000] print: 121
[@ 0.000] print: 122
[@ 0.000] print: 123
[@ 0.000] print: 124
[@ 0.000] print: 125
[@ 0.000] print: 126
[@ 0.000] print: 127
[@ 0.000] print: 17.1369
[@ 0.000] print: -1500
[@ 0.000] print: -1500
[@ 0.000] print: -13.726
